ClinGen/clincoded
# #2079: Disable submit buttons and adds error notification

**Steps to Test #2079**

1. Create a scorable piece of individual, experimental, or case-control evidence in a GDM
2. From the main gene-disease record page, click "View/Score" on your created piece of evidence
3. Add a score and click "Save"
4. Expect to see the "Save" button to disable and display a busy animation.

___

**Steps to Test Error Notification**
1. Repeat steps 1 and 2 above.
2. Go to your network tab in the Chrome dev tools and set the throttling from "Online" to "Offline" (to simulate an error occurring)
3. Add a score and click "Save"
4. Expect to see a red notification message show up in the top right corner of the page.
